src/lib/main.js

```diff
@@ -107,14 +107,14 @@ module.exports = async (argv, env) => {
         return -1;
       }
     } else {
-      instanceUrl = url.parse('http://admin:pass@localhost:5988');
+      instanceUrl = url.parse('http://admin:pass@localhost:5988/medic');
       info('Using default local url');
     }
   } else if (cmdArgs.instance) {
     const password = readline.question(`${emoji.key}  Password: `, { hideEchoBack: true });
     const instanceUsername = cmdArgs.user || 'admin';
     const encodedPassword = encodeURIComponent(password);
-    instanceUrl = url.parse(`https://${instanceUsername}:${encodedPassword}@${cmdArgs.instance}.medicmobile.org`);
+    instanceUrl = url.parse(`https://${instanceUsername}:${encodedPassword}@${cmdArgs.instance}.medicmobile.org/medic`);
   } else if (cmdArgs.url) {
     instanceUrl = url.parse(cmdArgs.url);
```

Comment:
I think we should continue to support people using URL-s without trailing db names and just use `medic` when no db name is provided. 
Also, while we're here, I think we should consider updating the code to not use the legacy URL API which is deprecated in Node 12 (https://nodejs.org/api/url.html#url_legacy_url_api), which `url.parse` is part of.